assets/queries-handler.js

```diff
@@ -8,6 +8,9 @@ const sortField = urlParams.get('sort_field');
 const sortOrder = urlParams.get('sort_order');
 const sortSelect = document.querySelector('.sort-select');
 let page = +urlParams.get('page[cod]');
+const dropdown = document.querySelector('.dropdown');
```

Comment:
The dropdown element can be used twice for example in the product page you should target the element with it's unique ID not the class